src/error.rs

```diff
@@ -37,6 +37,9 @@ pub enum CryptoError {
 
     /// Wrong nonce was provided during seal/unseal operation
     WrongNonceType,
+
+    /// The method is not implemented for the storage implementation
+    NotImplemented
```

Comment:
Can we add a

// TODO: Figure out a better way to do this

comment to this for future notes? The general idea is I can do a general text search on the codebase for `todo` and it'll yield all the things we don't really have a better idea for or don't want to deal with right now and aren't that important but could have better solutions. This is also a standard Rust thing:
https://doc.rust-lang.org/std/macro.todo.html

Reply:
ignoring and putting up another PR with an actual fix for this